src/api/AppMetadata.ts

```diff
@@ -10,6 +10,12 @@ export interface AppMetadata {
   /** The unique app name that can be used with the open and raiseIntent calls. */
   readonly name: string;
 
+  /** The unique application identifier located within a specific application directory instance.  */
```

Comment:
Add examples of a fully qualified appId to the comments

Reply:
Do we have such examples, or would we just be speculating?

Reply:
@kriswest I decided to go with consistency. I added an example to the comment based upon the documentation here https://fdc3.finos.org/docs/1.1/app-directory/discovery